### ISSUE TITLE
stdout is null when running from AWS Lambda

### ISSUE THREAD

**Issue by tminussi** [godaddy/kubernetes-client#490]
I am running into issues when running the client from within an AWS Lambda function. I get a 

> Failed to run cmd.

exception, which happens on line 44 of ```kubernetes-client/lib/auth-providers/cmd.js```

I have monkey patched the code and printed the exception accordingly. It turned out that line 42 was failing, because ```stdout``` is null when trying to JSON.parse it.
The offending line is:
```result = JSON.parse(output.stdout.toString('utf8'));```

Somehow Lambda is not populating (or is intentionally blocking) ```stdout```. 

My code is like this:

```
const client = new Client({
    config: config.fromKubeconfig('/tmp/config.dev'),
    version: '1.12',
  });
```

It runs just fine both locally and in Cloud9. It only fails in Lambda.

I am not sure this is something that can be fixed since, apparently, Lambda is the one screwing things up but I wanted to bring it up anyways since it may affect others trying to achieve the same thing as me in the future.

So, long story short, does anyone know why ```stdout``` is gone when running inside Lambda? If yes, is there anything we can do to work around it?

If somebody tells me how to fix it I can create a PR myself, but as of right now I have no clue where to start.

For reference only, here's how the object (partially) looks like in Cloud9:

![image](https://user-images.githubusercontent.com/3764756/58888729-243ca680-86e8-11e9-96c1-81f2fb2da52a.png)

And this is the same object (partially) in Lambda:

![image](https://user-images.githubusercontent.com/3764756/58888981-7bdb1200-86e8-11e9-93dc-9906f34e3d1c.png)

**Comment by CanKattwinkel** [comment]
Same for me. I'm loading a custom authentication file with `kubeconfig.loadFromString(creds);`. The script runs fine on my Ubuntu host but fails in Docker (node:10)

This is also related https://github.com/godaddy/kubernetes-client/issues/531
